about.md

```diff
@@ -488,7 +490,10 @@ users and contributors to participate in the project with respect and safety.
         {% for obj in page.sponsors %}
         <li>
             <a href="{{ obj.url }}">
-                <img src="{{ obj.src }}" alt="{{ obj.alt }}" />
+                <img src="{{ obj.src }}"
+                     width=300
+                     height=90
+                     alt="{{ obj.alt }}" />
```

Comment:
should this be lazy loaded? (and images elsewhere that are below the fold?)

Reply:
Yes, though we have a new volunteer in #532 who raised their hand to take this on. So I figured I'd let them have a shot and not scoop them.